core/pom.xml

```diff
@@ -135,6 +135,12 @@
       <version>${asciidoctorj-pdf.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <version>1.15</version>
+    </dependency>
+
 
     <dependency>
       <groupId>org.jboss.arquillian.junit</groupId>
```

Comment:
I got it but how would you do that? classloader.loadClass? I prefer to let this classpath manipulation with you ;). Can we create a separated issue for that?